code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -737,6 +737,10 @@
 	var/needs_update = M.mutations.len > 0 || M.disabilities > 0
 
 	if(needs_update)
+		var/mut
+		for(mut in M.mutations) //can be used to do specific things at mutations. values in _DEFINE/genetics.dm
```

Comment:
Could condense lines 740 to 742 down to
`if(DIZZY in M.mutations)`

Reply:
Good point. How do I change it without screwing everything up in this pull request?